pkg/formats/templates.go

```diff
@@ -20,6 +20,10 @@ var basicFunctions = template.FuncMap{
 	},
 	"split":    strings.Split,
 	"join":     strings.Join,
+	// strings.Title is deprecated since go 1.18
+	// However for our use case it is still fine. The recommended replacement
+	// is adding about 400kb binary size so lets keep using this for now.
+	//nolint:staticcheck
```

Comment:
Should we add a TODO or similar tag so we can more easily find this?

Reply:
I don't think it is a problem. Since it is in the stdlib it will stay there until golang decides to actively break code.

Reply:
better question why do we have this package? I thought we have this in c/common.
cc @jwhonce